suite-native/receive/src/screens/ReceiveModalScreen.tsx

```diff
@@ -0,0 +1,97 @@
+import { useSelector } from 'react-redux';
+
+import { CommonActions, RouteProp, useNavigation, useRoute } from '@react-navigation/native';
+import { G } from '@mobily/ts-belt';
+
+import { Text } from '@suite-native/atoms';
+import {
+    RootStackParamList,
+    RootStackRoutes,
+    Screen,
+    ScreenSubHeader,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { AccountsList } from '@suite-native/accounts';
+import { AccountKey, TokenAddress } from '@suite-common/wallet-types';
+import { Translation } from '@suite-native/intl';
+import {
+    AccountsRootState,
+    selectAccountLabel,
+    selectAccountNetworkSymbol,
+} from '@suite-common/wallet-core';
+import { selectEthereumAccountTokenSymbol } from '@suite-native/ethereum-tokens';
+
+import { ReceiveAccount } from '../components/ReceiveAccount';
+
+type ScreenSubHeaderContent = {
+    accountKey?: AccountKey;
+    tokenContract?: TokenAddress;
+};
+
+const ReceiveModalScreenSubHeader = ({ accountKey, tokenContract }: ScreenSubHeaderContent) => {
+    const accountLabel = useSelector((state: AccountsRootState) =>
+        selectAccountLabel(state, accountKey),
+    );
+    const networkSymbol = useSelector((state: AccountsRootState) =>
+        selectAccountNetworkSymbol(state, accountKey),
+    );
+    const tokenSymbol = useSelector((state: AccountsRootState) =>
+        selectEthereumAccountTokenSymbol(state, accountKey, tokenContract),
```

Comment:
i suggest to use `ethereumTokenSymbol` because there probably will be other tokens once Solana is merged

Reply:
fixed in 8af235d6e7